docs/contexts/traceability-v1.jsonld

```diff
@@ -14,16 +14,16 @@
       "@id": "https://w3id.org/traceability/AgInspectionReport",
       "@context": {
         "facility": {
-          "@id": "https://schema.org/Place"
+          "@id": "https://w3id.org/traceability/schemas/Place.json"
```

Comment:
this is not a valid definition.... see https://github.com/w3c-ccg/citizenship-vocab/blob/main/contexts/citizenship-v1.jsonld#L42

https://w3id.org/citizenship#commuterClassification

Note that `@id` should point to _human readable_ documentation for the term.

For example:

https://w3id.org/traceability/#facility-identifier

https://w3id.org/traceability/#place -> here was want this to be "https://schema.org/Place" since that is required to "use schema.org vocabulary".

Reply:
left a comment in https://github.com/w3c-ccg/traceability-vocab/issues/53